src/olympia/discovery/admin.py

```diff
@@ -27,12 +28,58 @@ def clean(self, value):
         return super(SlugOrPkChoiceField, self).clean(value)
 
 
+class PositionFilter(admin.SimpleListFilter):
+    # Title for the filter section.
+    title = 'presence in Disco Pane editorial content'
+
+    # Parameter for the filter that will be used in the URL query.
+    parameter_name = 'position'
+
+    # Database field to use.
+    db_field = 'position'
+
+    def lookups(self, request, model_admin):
+        """
+        Returns a list of tuples.
+        # - The first element in each tuple is the coded value for the option
+        #   that will appear in the URL query.
+        # - The second element is the human-readable name for the option that
+        #   will appear
+        in the right sidebar.
+        """
+        return (
+            ('yes', 'Yes'),
+            ('no', 'No'),
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Returns the filtered queryset based on the value provided in the query
+        string and retrievable via `self.value()`.
+        """
+        # Compare the requested value (either 'on' or 'off')
+        # to decide how to filter the queryset.
+        if self.value() == 'yes':
+            return queryset.filter(**{self.db_field + '__gt': 0})
+        if self.value() == 'no':
+            return queryset.filter(
+                Q(**{self.db_field: 0}) | Q(**{self.db_field: None}))
+
+
+class PositionChinaFilter(PositionFilter):
+    title = 'presence in Disco Pane editorial content (China edition)'
+    parameter_name = 'position_china'
+    db_field = 'position'
```

Comment:
Already defined in the parent Class

Reply:
It's actually wrong, it should be `db_field = 'position_china'` (I haven't added tests for this yet)

Reply:
maybe you don't need both `parameter_name` and `db_field` then if they're the same values.